konduit-serving-test/src/test/java/ai/konduit/serving/verticles/python/custom/PythonFloatInputTest.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
What is the difference between this class and the `PythonFloatInputOperationTest.java`?